tests/python/gaia-ui-tests/gaiatest/apps/system/app.py

```diff
@@ -35,7 +35,7 @@ def wait_for_status_bar_displayed(self):
         self.wait_for_element_displayed(*self._status_bar_locator)
 
     def wait_for_notification_toaster_displayed(self, timeout=10, message=None):
-        self.wait_for_element_displayed(*self._notification_toaster_locator)
+        self.wait_for_element_displayed(*self._notification_toaster_locator, timeout=timeout)
 
     def wait_for_notification_toaster_not_displayed(self, timeout=10):
         self.wait_for_element_not_displayed(*self._notification_toaster_locator)
```

Comment:
Same problem here. I know you didn't introduce that at all, but it'd be good to A) give the two methods symmetry, so if you're going to add message to one and B) actually pass the parameters along.

If you were willing to at least fix B here since you're in the area, that'd be awesome.